lib/common/client.ts

```diff
@@ -0,0 +1,29 @@
+import { useState, useEffect } from "react";
+import type { TUseStoryblokState } from "../types";
+import { registerStoryblokBridge } from "@storyblok/js";
+
+export const useStoryblokState: TUseStoryblokState = (
+  initialStory = null,
+  bridgeOptions = {}
+) => {
+  let [story, setStory] = useState(initialStory);
+
+  const isBridgeEnable =
+    typeof window !== "undefined" &&
+    typeof window.storyblokRegisterEvent !== "undefined";
+
+  if (!isBridgeEnable || !initialStory) {
+    return initialStory;
+  }
+
+  useEffect(() => {
+    setStory(initialStory);
```

Comment:
I think you don't need to set again the initialStory in the useEffect, you already init at the beginning with useState